submodules/faxbox/views/edit.html

```diff
@@ -73,7 +73,19 @@ <h3>{{ i18n.callflows.faxbox.notification_emails }}</h3>
 				<div class="clearfix">
 					<label for="caller_id">{{ i18n.callflows.faxbox.caller_id }}</label>
 					<div class="input">
-						<input class="span4" id="caller_id" name="caller_id" type="text" placeholder="{{ i18n.callflows.faxbox.caller_id_placeholder }}" value="{{faxbox.caller_id}}" rel="popover" data-content="{{ i18n.callflows.faxbox.caller_id_data_content }}"/>
+						{{#if ensure_valid_caller_id}}
+							<span rel="popover" data-content="{{ i18n.callflows.faxbox.caller_id_data_content }}">
+								<select name="caller_id" id="caller_id" class="medium">
+								{{#select faxbox.caller_id}}
+									{{#each phone_numbers}}
+										<option value="{{id}}" id="{{id}}">{{label}}</option>
```

Comment:
To format phone numbers and make them more readable for users, you can use our custom Handlebars helper ([doc](https://docs.2600hz.com/ui/docs/handlebars/#formatphonenumber)):
```suggestion
										<option value="{{id}}" id="{{id}}">
											{{formatPhoneNumber label}}
										</option>
```
Also, I don't see why each `<option>` has an `id` here 🤔

Reply:
great, thanks, the `id` prop was just a copy/paste :(